bigquery/src/test/java/com/google/cloud/hadoop/io/bigquery/AvroRecordReaderTest.java

```diff
@@ -46,10 +46,23 @@
   @Before
   public void setup() throws IOException {
     Schema schema =
-        SchemaBuilder.record("BigQueryRecord").fields()
-            .name("key").type().stringBuilder().endString().noDefault()
-            .name("value1").type().stringBuilder().endString().noDefault()
-            .name("value2").type().intBuilder().endInt().noDefault()
+        SchemaBuilder.record("BigQueryRecord")
+            .fields()
+            .name("key")
+            .type()
+            .stringBuilder()
+            .endString()
+            .noDefault()
+            .name("value1")
+            .type()
+            .stringBuilder()
+            .endString()
+            .noDefault()
+            .name("value2")
+            .type()
+            .intBuilder()
+            .endInt()
+            .noDefault()
```

Comment:
Revert - probably better to turn off formatting for this code

Reply:
The frequency of changes to this part of the code is quite low, we might be better off formatting one off.

Turning off formatting at a file level seems like an overkill

Reply:
It can be disabled for a snippet: https://github.com/diffplug/spotless/tree/main/plugin-maven#spotlessoff-and-spotlesson

Reply:
ack